lib/widgets/theme.dart

```diff
@@ -114,13 +114,14 @@ class DesignVariables extends ThemeExtension<DesignVariables> {
       mainBackground: const Color(0xfff0f0f0),
       title: const Color(0xff1a1a1a),
       channelColorSwatches: ChannelColorSwatches.light,
-      atMentionMarker: const HSLColor.fromAHSL(0.5, 0, 0, 0.2).toColor(),
+      atMentionMarker: const HSLColor.fromAHSL(0.7, 0, 0, 0.2).toColor(),
+      mutedAtMentionMarker: const HSLColor.fromAHSL(0.5, 0, 0, 0.2).toColor(),
```

Comment:
Could you explain how you chose both of these colors? I see `atMentionMarker was changed from its previous value, but I don't understand why.

Reply:
Well I've tried to copy the values from the web app.. here is my inspector values:
for unmuted mentions and mentions in inbox screen
`hsl(0, 0, 0.2)` or `#333` with an opacity `0.7`


And for muted mentions:

`hsla(0, 0%, 20%, .5)` with an opacity `0.7`

Which could be simplified as: `hsla(0, 0%, 20%, 0.35)`

Please let me know if I am missing something.

Reply:
It will probably be helpful to link (in the commit message) the lines containing the CSS styles in effect.